### PR TITLE
fix: issues with menu component

### PR DESCRIPTION
**Pull request by apvale**
## Description of the change

Fixes issues with the Menu component and adds minW and maxW as shown in Figma styles. 

Before: 

image

After:

image

## Testing the change

- [ ] Write your testing instructions here

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Non-Breaking Change (change to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached.

### CODE REVIEW

File: .changeset/chilly-geese-reflect.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+"@localyze-pluto/components": patch
+---
+
+[Menu] Fix issues on Menu component
+
+- Remove extra padding
+- Use outline instead of border
```
Comment (nyan07) [inline .changeset/chilly-geese-reflect.md:8]: I would describe what it fixes, not what the solution was.

**Review by nyan07 — APPROVED** [review]

File: packages/components/src/components/Menu/Menu.tsx [hunk 2]
```diff
@@ -50,14 +50,15 @@ const Menu = React.forwardRef<HTMLButtonElement, MenuProps>(
...
-            borderColor="colorBorderPrimary"
+            backgroundColor="colorBackground"
             borderRadius="borderRadius20"
-            borderStyle="borderStyleSolid"
-            borderWidth="borderWidth10"
             display="flex"
             flexDirection="column"
-            paddingBottom="space20"
-            paddingTop="space20"
+            maxWidth="280px"
```
Comment (rssilva) [inline packages/components/src/components/Menu/Menu.tsx:57]: Late to the party here but shouldn't this maxWidth be configurable as a prop? If we have text a bit bigger as option will the line break?

Comment (apvale) [reply packages/components/src/components/Menu/Menu.tsx:57]: Thank you for the heads up, Rafa. 🙏🏻 
I spoke with Hanna, who said the option should break the line when reaching the maxWidth. 

image

**Comment by apvale** [comment]
@nyan07 @rssilva Can you check again the PR after I changed a bit the Menu component? https://github.com/Localitos/pluto/pull/1108/commits/30f0f1435a6bac572bee0a6794772ee6b867313d
I hope the changeset is improved. But I noticed that the item wasn't triggering the onClick, only if I clicked exactly in the button. So, I moved the trigger to the whole menu item.

**Pull request merged**